### DISCUSSION TITLE
[bug] Property added under accounts is throwing an error while integrating in to .net core solution

### DISCUSSION THREAD

**Discussion by kkjithin** [YetiForceCompany/YetiForceCRM#15624] [Feature improvement]
We are adding the below property on the accounts section. What happens is for some accounts this property is coming with value but for some accounts its not reflecting on the model even the fieldname.
![image](https://user-images.githubusercontent.com/74819613/148922611-933dcb52-5ac4-4bcb-aa42-e31c27726c33.png)
![image](https://user-images.githubusercontent.com/74819613/148923376-4e0d35af-82da-41e3-ab93-735cf28b64c8.png)
![image](https://user-images.githubusercontent.com/74819613/148923439-e9956dac-2ac4-457d-a705-bd5d46f9a2c5.png)
![image](https://user-images.githubusercontent.com/74819613/148923519-ccf75f4e-f184-4c19-a0e8-438d4818e0d9.png)

**Comment by mariuszkrzaczkowski** [comment]
I don't understand your problem, describe the API problem in more detail
